node/resolvers/search/index.ts

```diff
@@ -280,6 +286,37 @@ const getSellers = async (
   )
 
   return result?.find((region: Region) => region.id === regionId)?.sellers
+  }
+
+const buildSpecificationFiltersAsFacets = (specificationFilters: string[]): SelectedFacet[]  => {
+  const specificationFilterFacets: SelectedFacet[] = []
+  specificationFilters.map((specificationFilter: string) => {
+    const [key, value] = specificationFilter.split(":")
+    specificationFilterFacets.push({key: key, value: value})
+  })
+  return specificationFilterFacets
+}
```

Comment:
```suggestion
const buildSpecificationFiltersAsFacets = (specificationFilters: string[]): SelectedFacet[]  =>
  specificationFilters.map((specificationFilter: string) => {
    const [key, value] = specificationFilter.split(":")
    return {key, value}
  })
```

if you just return the map this way you don't need to create a new array :)